components/MyKeyboard/index.js

```diff
@@ -19,7 +22,24 @@ const StyledSendSMSContainer = styled.div`
 
 const StyledPseudoInput = styled.p`
   width: 80%;
-  background-color: white;
+  background-color: ${(props) => {
+    if (props.theme === "light") {
+      return "white";
+    } else if (props.theme === "dark") {
+      return "white";
+    } else if (props.theme === "custom") {
+      return "white";
+    }
+  }};
+  color: ${(props) => {
+    if (props.theme === "light") {
+      return "black";
+    } else if (props.theme === "dark") {
+      return "black";
+    } else if (props.theme === "custom") {
+      return "black";
+    }
+  }};
```

Comment:
Here you forgot to adjust the actual assigned colors, I guess? Same for `background-color` above.